src/status_im/ui/screens/peers_stats.cljs

```diff
@@ -0,0 +1,15 @@
+(ns status-im.ui.screens.peers-stats
+  (:require [quo.react-native :as quo.react-native]
+            [status-im.ui.components.react :as react]
+            [status-im.i18n.i18n :as i18n]
+            [re-frame.core :as re-frame.core]))
+
+(defn peers-stats []
+  (let [peers-count @(re-frame.core/subscribe [:peers-count])]
```

Comment:
[Guidelines] We're using `utils.re-frame/sub` instead of `re-frame.core/subscribe`.

Reply:
fixed, thanks